From a3373aa9b0a5bf473357b337f5f0944552831262 Mon Sep 17 00:00:00 2001
From: Sebastian Pipping <sebastian@pipping.org>
Date: Wed, 3 Jan 2018 19:17:22 +0100
Subject: [PATCH] Fix start-up crash from broken PyGObject dependency

$ safeeyes
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.5/safeeyes", line 6, in <module>
    from pkg_resources import load_entry_point
  [..]
  File "/usr/lib64/python3.5/site-packages/pkg_resources/__init__.py", line 870, in resolve
    raise DistributionNotFound(req, requirers)
pkg_resources.DistributionNotFound: The 'gi' distribution was not found and is required by safeeyes

$ python3.5 -c 'import gi; print(gi.__file__)'
/usr/lib64/python3.5/site-packages/gi/__init__.py
---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index e6e4ef1..5dff037 100644
--- a/setup.py
+++ b/setup.py
@@ -6,8 +6,7 @@ import setuptools
 requires = [
     'babel',
-    'dbus',
-    'gi',
     'psutil',
+    'PyGObject',
     'python-xlib'
 ]
 
-- 
2.15.0

